CMakeLists.txt

```diff
@@ -4,8 +4,10 @@ project(libhsplasma)
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
+set(CMAKE_CXX_VISIBILITY_PRESET hidden)
+set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
```

Comment:
I feel like these should be set per-target so as to not conflict with projects that may want to include HSPlasma as a sub-project (eg via `FetchContent_MakeAvailable()`). But, seeing as how we already clobber the `CMAKE_CXX_STANDARD`, I don't think this is a deal breaker.